Google.Api.Generator.Tests/ProtoTests/UnitTests/Testing.UnitTests.Tests/UnitTestsClientTest.g.cs

```diff
@@ -0,0 +1,108 @@
+﻿using gax = Google.Api.Gax;
+using proto = Google.Protobuf;
+using grpccore = Grpc.Core;
+using moq = Moq;
+using xunit = Xunit;
+
+namespace Testing.UnitTests
+{
+    /// <summary>Generated unit tests.</summary>
+    public sealed class GeneratedUnitTestsTest
+    {
+        [xunit::FactAttribute]
+        public void MethodValuesRequestObject()
+        {
+            moq::Mock<UnitTests.UnitTestsClient> mockGrpcClient = new moq::Mock<UnitTests.UnitTestsClient>(moq::MockBehavior.Strict);
+            ValuesRequest request = new ValuesRequest
```

Comment:
I'm not sure I see the value of setting all the properties here. The mock's just validating that it's receiving an equal object, and in fact it will be a reference to the *same* object, right?

Reply:
Yes, you're right.
However, this is the "one true way" of writing these tests. And I suppose is more helpful if someone is looking at the tests to learn from them. And I feel it's probably the best thing to do, even though it's pointless. Like this entire test.